frontend/src/component/playground/Playground/PlaygroundForm/PlaygroundCodeFieldset/PlaygroundCodeFieldset.tsx

```diff
@@ -103,6 +107,27 @@ export const PlaygroundCodeFieldset: VFC<IPlaygroundCodeFieldsetProps> = ({
         }
     };
 
+    const handleAutocompleteOnChange = (e: FormEvent, newValue: any) => {
+        if (typeof newValue === 'string') {
+            return setContextValue(newValue);
+        }
+        if (Array.isArray(newValue)) {
+            const temp =
+                (newValue as Array<string>).length > 1
+                    ? newValue.join(',')
+                    : newValue[0];
+            return setContextValue(temp);
+        }
+    };
+
+    const resolveValue = isAdvancedPlayground
```

Comment:
I find this part difficult to read with ternay inside ternary. I'd extract and intention revealing function that I can call and I'd unit test it to explain the purpose.